marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -29,14 +29,25 @@ describe('RTL Source-to-entity map tests', () => {
     });
 
     test('RTL tests with source data',  () => {
-        const { getByTestId,  getByText, queryByText, rerender} = render(<SourceToEntityMap {...data.mapProps} mappingVisible={true}/>);
+        const { getByTestId,  getByText, queryByText, rerender } = render(<SourceToEntityMap {...data.mapProps} mappingVisible={true}/>);
         expect(getByText('Source Data')).toBeInTheDocument();
         expect(getByText('proteinId')).toBeInTheDocument();
+        expect(getByText('emptyString')).toBeInTheDocument();
+        expect(getByText('nullValue')).toBeInTheDocument();
+        expect(getByText('numberValue')).toBeInTheDocument();
+        expect(getByText('booleanValue')).toBeInTheDocument();
         expect(getByTestId("entityContainer")).toBeInTheDocument();
         expect(getByTestId("srcContainer")).toBeInTheDocument();
         expect(getByTestId("srcContainer")).toHaveClass("sourceContainer");
         expect(getByText('Entity: Person')).toBeInTheDocument();
         expect(getByText('Test')).toBeEnabled();
+
+        // Check datatype class names for source values
+        expect(getByTestId("emptyString-srcValue").children[0].className.includes('datatype-string')).toBe(true);
+        expect(getByTestId("nullValue-srcValue").children[0].className.includes('datatype-null')).toBe(true);
+        expect(getByTestId("numberValue-srcValue").children[0].className.includes('datatype-number')).toBe(true);
+        expect(getByTestId("booleanValue-srcValue").children[0].className.includes('datatype-boolean')).toBe(true);
```

Comment:
@wooldridge can we pls also add a test for white space here.

Reply:
Thanks, added test for whitespace with latest force-push.